python/cudf_kafka/LICENSE

```diff
@@ -0,0 +1 @@
+../../LICENSE
```

Comment:
Question: Just double-checking since it's a limitation of the GH GUI that's bitten us before, this file is a symlink and not an actual file containing `../../LICENSE` as text, right?

Reply:
Yes, it's a symlink. Same for the README.